ga4gh/client.py

```diff
@@ -15,13 +15,26 @@
 requests.packages.urllib3.disable_warnings()
 
 
-class HTTPClient(object):
-
-    workaroundGoogle = 'google'
-
+class HttpClient(object):
```

Comment:
I'd like to advocate disobeying this part of PEP8.  Noun boundaries are difficult to parse with all-caps acronyms, especially with a variable name that has multiple acronyms.  For instance:

HttpClient is easier to read than HTTPClient
HttpApiClient is easier to read than HTTPAPIClient